internal/pkg/flink/internal/store/store_utils_test.go

```diff
@@ -141,30 +141,30 @@ func TestProcessResetStatement(t *testing.T) {
 		s.Properties["timeout"] = "30"
 		result, _ := s.processResetStatement("reset")
 		assert.EqualValues(t, types.COMPLETED, result.Status)
-		assert.Equal(t, "Configuration has been reset successfully.", result.StatusDetail)
+		assert.Equal(t, "Configuration has been reset successfully", result.StatusDetail)
 		assert.Nil(t, result.StatementResults)
 	})
 
 	t.Run("should return an error message if configKey does not exist", func(t *testing.T) {
 		result, err := s.processResetStatement("reset 'location'")
 		assert.NotNil(t, err)
-		assert.Equal(t, "Error: Config key \"location\" is currently not set.", err.Error())
+		assert.Equal(t, `Error: Config key "location" is not set.`, err.Error())
```

Comment:
```suggestion
		assert.Equal(t, `Error: configuration key "location" is not set.`, err.Error())
```

Reply:
changed config to configuration in all places!